infra/base-images/base-runner/run_fuzzer

```diff
@@ -104,4 +104,14 @@ else
 fi
 
 echo $CMD_LINE
+
+# Unset OUT so the fuzz target can't rely on it.
+INITIAL_OUT=$OUT
+unset OUT
 bash -c "$CMD_LINE"
+RETURNCODE=$?
```

Comment:
is this returncode needed ? since script is run as -eu

Reply:
Guess not. Done.